package.json

```diff
@@ -3,15 +3,22 @@
 	"version": "0.1.0",
 	"private": true,
 	"scripts": {
-		"serve": "vue-cli-service serve",
-		"build": "vue-cli-service build",
+		"serve": "vue-cli-service serve  --mode sample",
```

Comment:
what does `--mode sample` do? It doesn't appear to be documented on the Vue site.

Reply:
This is applied because of the sample.env file been used